rir/src/common.h

```diff
@@ -12,9 +12,11 @@ extern void printBacktrace();
 
 #ifdef ENABLE_SLOWASSERT
 #define SLOWASSERT(what) assert(what)
+__attribute__((unused)) static bool IS_SLOWASSERT = true;
```

Comment:
that's not a good idea, to basically duplicate this. either we make it a static variable or a macro, but not both! also I think it's not needed, see comment below